contract/src/postal-service.contract.js

```diff
@@ -38,9 +40,19 @@ export const start = zcf => {
    */
   const sendTo = (addr, pmt) => E(getDepositFacet(addr)).receive(pmt);
 
-  /** @param {string} recipient */
-  const makeSendInvitation = recipient => {
+  /**
+   * @param {string} recipient
+   * @param {Issuer[]} issuers
+   */
+  const makeSendInvitation = (recipient, issuers) => {
     assert.typeof(recipient, 'string');
+    mustMatch(issuers, M.arrayOf(IssuerShape));
```

Comment:
Did you consider making the `issuers` arg optional?

Reply:
The UI is always just going to send the issuers anyway so it doesn't seem worth the complexity. We'd have to query and check which issuers are already added first, I'm not sure if they're published anywhere either.